modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -29,13 +24,25 @@ def show
 
     private
 
-    def fetch_or_error_local_claim_id
-      claim = ClaimsApi::AutoEstablishedClaim.find_by(id: params[:id])
+    def find_claim
+      claim = ClaimsApi::AutoEstablishedClaim.find_by(id: params[:id]) # , source: source_name)
```

Comment:
@bastosmichael not sure if you still wanted to try to add in the `source` property of if that can be removed now

Reply:
Can you remove it from just the find_by methods for now and I'll create a ticket to address this later as technical debt